skimage/measure/_moments.py

```diff
@@ -352,6 +360,14 @@ def centroid(image):
     -------
     center : tuple of float, length ``image.ndim``
         The centroid of the (nonzero) pixels in ``image``.
+
+    Examples
+    --------
+    image = np.zeros((20, 20), dtype=np.double)
```

Comment:
This line currently does not pass the CI. I believe it should be changed to:
```suggestion
    >>> image = np.zeros((20, 20), dtype=np.double)
```